lm_eval/models/anthropic_llms.py

```diff
@@ -92,20 +116,28 @@ def greedy_until(self, requests):
 
         res = []
         for request in tqdm(requests):
-            inp = request[0]
-            request_args = request[1]
-            until = request_args["until"]
-            response = anthropic_completion(
-                client=self.client,
-                model=self.model,
-                prompt=inp,
-                max_tokens_to_sample=self.max_gen_toks,
-                temperature=0.0,  # TODO: implement non-greedy sampling for Anthropic
-                stop=until,
-            )
-            res.append(response)
-
-            self.cache_hook.add_partial("greedy_until", request, response)
+            try:
+                inp = request[0]
+                request_args = request[1]
+                until = request_args["until"]
+                response = anthropic_completion(
+                    client=self.client,
+                    model=self.model,
+                    prompt=inp,
+                    max_tokens_to_sample=self.max_tokens_to_sample,
+                    temperature=self.temperature,  # TODO: implement non-greedy sampling for Anthropic
```

Comment:
`temperature` should be passed from `request_args['temperature']` and be 0.0 otherwise